src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/customer.html.twig

```diff
@@ -135,33 +146,35 @@
         <a class="btn btn-outline-secondary js-details-customer-btn" href="">
           {{ 'Details'|trans({}, 'Admin.Global') }}
         </a>
-        <button type="button" class="btn btn-primary float-right js-choose-customer-btn">
+        <button type="button" class="btn btn-outline-primary float-right js-choose-customer-btn">
           {{ 'Choose'|trans({}, 'Admin.Actions') }}
         </button>
-        <button type="button" class="btn btn-primary float-right js-change-customer-btn d-none">
+        <button type="button" class="btn btn-outline-primary float-right js-change-customer-btn d-none">
           {{ 'Change'|trans({}, 'Admin.Actions') }}
         </button>
       </div>
     </div>
   </div>
 </div>
 
-<div id="customer-search-empty-result-template" class="d-none">
-  <div class="col">
-    <div class="alert alert-warning" role="alert">
-      <p class="alert-text">{{ 'No customers found'|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
-    </div>
-  </div>
-</div>
-
 <script id="customer-carts-table-row-template" type="text/template">
   <tr>
     <td class="js-cart-id"></td>
     <td class="js-cart-date"></td>
     <td class="js-cart-total"></td>
     <td class="text-right">
-      <a class="btn btn-primary js-cart-details-btn" href="">{{ 'Details'|trans({}, 'Admin.Global') }}</a>
-      <button class="btn btn-primary js-use-cart-btn">{{ 'Use'|trans({}, 'Admin.Orderscustomers.Feature') }}</button>
+      <a
```

Comment:
Same thing here, why not use a button instead of an empty link?

Reply:
Answered above 😊